diff --git a/node_modules/@nativescript/canvas-media/platforms/android/canvas_media.aar b/node_modules/@nativescript/canvas-media/platforms/android/canvas_media.aar
index f6a519f..d123294 100644
Binary files a/node_modules/@nativescript/canvas-media/platforms/android/canvas_media.aar and b/node_modules/@nativescript/canvas-media/platforms/android/canvas_media.aar differ
diff --git a/node_modules/@nativescript/canvas-media/video/index.android.js b/node_modules/@nativescript/canvas-media/video/index.android.js
index 5a5b5cc..740ee29 100644
--- a/node_modules/@nativescript/canvas-media/video/index.android.js
+++ b/node_modules/@nativescript/canvas-media/video/index.android.js
@@ -67,7 +67,7 @@ export class Video extends VideoBase {
     getCurrentFrame(context) {
         const ctx = arguments[1];
         const flipY = ctx._flipY;
-        const ptr = ctx._canvas._canvas.getNativeGL();
+        const ptr = ctx._canvas._canvas.getNativeContext();
         this._instance.getCurrentFrame(!!this.isLoaded, ptr, flipY, arguments[4], arguments[5]);
     }
     play() {
